docs/golden_metrics.md

```diff
@@ -47,7 +48,29 @@ memoryUsage:
 | select    |      Yes      |  | Provide the field and function you want to display in the metric. You must only provide one field, but you can do aggregations, sums, etc. Always name the fields to make it easier to read. e.g. `sum((provider.httpCodeElb4XXCount.Sum OR 0) + (provider.httpCodeElb5XXCount.Sum OR 0)) AS 'Errors'`|
 | from |   No    | `Metric` | Choose where your metric gathers the information from |
 | where |   No    | empty string | In the case you need a more granular WHERE clause added into the query you can use this field. e.g. `provider='Alb'` |
-| facet |   No    | `entity.name` | Which field to use when doing a FACET. This field will be used when displaying the metric for several entities |
+| facet |   No    | empty string | An extra facet by a specific field to be added to the default facet by entityName |
 | eventId |   No    | `entity.guid` | The event attribute used to filter the entity. We recommend to use the default `entity.guid` which is generated automatically as part of the entity synthesis. |
+| eventName |   No    | `entity.name` | The name of the field in the event that references the entity name. By default `entity.guid` which is generated automatically as part of the entity synthesis. |
 
+### Roll-up entities
 
+In the cases that the entity type can be ingested from different sources and you need to provide a different query implementation you can use `queries` instead of `query`
+
+```yaml
+memoryUsage:
+  title: "A title explaining what the user is seeing (unit displayed in the dashboard)"
+  queries:
+    prometheus:
+      select: average(field)
+      from: PrometheusSample
+    newRelic:
+      select: average(nrField)
+      from: NewRelicSample
+```
+
+In this example `prometheus` and `newRelic` are the values the entity must have in the `instrumentation.provider` tag.
+The first tag value that matches with the entity will be the one used to build the queries.
+
+Is also important to note that the semantics of the queries should match between each implementation. This includes things like average vs counts, units and other details.
+
+If no rule matches the `newRelic` one will be used.
```

Comment:
```suggestion
If no rule matches the telemetry, the `newRelic` one will be used.
```

Reply:
hmm this is not really matching against telemetry but the tags the entity has.

Reply:
True, can we change it to the tags then? (I just felt that the phrase felt a bit weird when I read it)